cli/src/main.rs

```diff
@@ -62,7 +66,7 @@ pub async fn main() {
   let anycloud_agz = base64::encode(include_bytes!("../alan/anycloud.agz"));
   let app = App::new(crate_name!())
     .version(crate_version!())
-    .about("AnyCloud is a Lambda alternative that works with multiple cloud provider.")
+    .about(concat!("alan ", alan_version!(), "\n", env!("CARGO_PKG_DESCRIPTION")))
```

Comment:
nit can we keep the format of your original message, so `concat!(env!("CARGO_PKG_DESCRIPTION"), ".\nCurrent alan version: ", alan_version!())`

Reply:
ok!